e2e/widenotation/testUsingRw.py

```diff
@@ -0,0 +1,79 @@
+import time
+import unittest
+
+from appium import webdriver
+from appium.options.android import UiAutomator2Options
+from appium.webdriver.common.appiumby import AppiumBy
+
+options = UiAutomator2Options()
+options.platformName = 'Android'
+options.device_name = 'Pixel_3a_API_34_extension_level_7_x86_64'
+options.app = 'D:/STV/HushKeyboard/app/build/outputs/apk/debug/app-debug.apk'
+appium_server_url = 'http://127.0.0.1:4723/wd/hub'
+
+class TestAppium(unittest.TestCase):
```

Comment:
Please help me rename the test case.

```suggestion
class TestWideNotationInputWithWideNotation(unittest.TestCase):
```